core/foundation/src/TClassEdit.cxx

```diff
@@ -30,6 +30,17 @@
 
 namespace {
    static TClassEdit::TInterpreterLookupHelper *gInterpreterHelper = 0;
+
+   template <typename T>
```

Comment:
Why template here?

Reply:
because it a wrapper around the actual type of the static variable.